src/main/java/io/rcktapp/api/service/Configurator.java

```diff
@@ -486,7 +486,9 @@ Config findConfig(Iterator<Map.Entry<Object, Object>> propertySource) throws IOE
          }
       }
 
-      config.files.add("ADDITIONAL PROPERTIES");
+      if (propertySource.hasNext()) {
+         config.files.add("ADDITIONAL ITERATOR PROPERTIES");
+      }
```

Comment:
Needed to make this conditional... no point in adding it to the list of config files if we didn't actually add anything.